static/src/javascripts/projects/commercial/modules/gallery-adverts.js

```diff
@@ -64,7 +55,15 @@ define([
                 item.anchor.appendChild(item.adSlot);
             }
 
-    });
+        });
+
+    }
+
+    function getSlotNameForMobile(index) {
```

Comment:
Yes! Whilst I was doing the change, I thought exactly the same. This should be in `create-slot`, and naming the slot at the calling site should be optional. Will change next 😃